packages/hub/bootstrap-schema.js

```diff
@@ -693,6 +813,55 @@ const models = [
       'field-type': '@cardstack/core-types::object'
     }
   },
+  {
+    type: 'fields',
+    id: 'member-name',
```

Comment:
What's the deal with the `member-name` field on `content-types` and `fields`?

Is it derived state, or something people are really allowed to set in their schema.js?

If it's internal derived state, it doesn't belong in bootstrap schema, and would only be a property on the actual `Field` and `ContentType` classes we use internally.

If it's the public API for setting the name of a field or content-type, then it should be the *only* public API for setting them. And we could call it `name` instead (remember, even the bootstrap schema will now be scoped! Field names in here should not collide with any user fields).

Reply:
Yeah, it;s the public API for setting the name of the field. I was hedging on using the word "name" for this attribute, and wanted to be a bit more descriptive (JSON:API itself calls this thing the member name). But I can go back to using `name`.